File: PHP/Model/Professor.php
<?php
//session_start();

require_once 'Usuario.php';
  class Professor extends Usuario{
   private $id; 
   private $rgmProfessor;
   
    public function __construct(){
       parent::__construct();
    }   
  
     public function getId(){
        return $this->id;
    }

    public function setId($id){
        $this->id = $id;
    }

    public function getRgmProfessor(){
        return $this->rgmProfessor;
    }

    public function setRgmProfessor($rgmProfessor){

        $this->rgmProfessor = $rgmProfessor;
    }

    public function __toString(){
    
        return "Rgm Professor: {$this->rgmProfessor}". parent::__toString();
    }

  }


?>

File: JAVA/Model/Professor.java
package Model;
public class Professor extends Usuario {
    private int id;
    private String rgmProfessor;

    public Professor(){
        super();
    }
    
    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }
    public String getrgmProfessor() {
        return rgmProfessor;
    }

    public void setRgmProfessor(String rgmProfessor) {
        this.rgmProfessor = rgmProfessor;
    }

    @Override
    public String toString() {
        return super.toString() +  " RgmProfessor=" + rgmProfessor + "\n";
    }

     public String stringSql(){
        return "INSERT INTO professor (id, rgmProfessor, nome, email, senha)"
        + "VALUES('"+this.id+"', '"+this.rgmProfessor+"','"+getNome()+"', '"+getEmail()+"', '"+getSenha()+"');\n";
    }


    

    
    


}
